iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Switch.java

```diff
@@ -53,4 +56,44 @@
      * @param retained the retain status of the switch
      */
     void setRetained(boolean retained);
+
+    default List<OperationalLimits> getOperationalLimits1() {
+        return Collections.emptyList();
+    }
+
+    default <L extends OperationalLimits> L getOperationalLimits1(LimitType limitType, Class<L> limitClazz) {
+        return null;
+    }
+
+    default CurrentLimitsAdder newCurrentLimits1() {
+        throw new UnsupportedOperationException();
```

Comment:
Mmmmm I think that we have a problem here. Do you have any idea of the solution ?

Reply:
Do we have a side for switches ?

Reply:
We don't have "side" object per see, but we discriminate between node1 and node2 (or bus1 and bus2) of a switch